src/integrationTest/java/uk/gov/hmcts/reform/pip/subscription/management/controllers/SubscriptionControllerTests.java

```diff
@@ -727,6 +728,85 @@ void testBuildDeletedArtefactSubscribersReturnsAccepted() throws Exception {
         );
     }
 
+    @Test
+    void testBuildBulkDeletedSubscribersReturnsAccepted() throws Exception {
+        MvcResult caseSubscription =
+            mvc.perform(setupMockSubscription(CASE_ID, SearchType.CASE_ID, VALID_USER_ID)).andReturn();
+        MvcResult locationSubscription =
+            mvc.perform(setupMockSubscription(LOCATION_ID, SearchType.LOCATION_ID, UUID_STRING)).andReturn();
+
+        String caseSubscriptionId = getSubscriptionId(caseSubscription.getResponse().getContentAsString());
+        String locationSubscriptionId = getSubscriptionId(locationSubscription.getResponse().getContentAsString());
+
+        String subscriptionIdRequest = "[\"" + caseSubscriptionId + "\","
+            + "\"" + locationSubscriptionId + "\"]";
+
+
+        MvcResult deleteResponse = mvc.perform(delete(DELETED_BULK_SUBSCRIPTION_PATH)
+                                                   .contentType(MediaType.APPLICATION_JSON)
+                                                   .content(subscriptionIdRequest))
+            .andExpect(status().isOk()).andReturn();
+
+        assertEquals(String.format("Subscription(s) with ID %s deleted",
+                                   caseSubscriptionId + ", " + locationSubscriptionId),
+                     deleteResponse.getResponse().getContentAsString(), RESPONSE_MATCH
+        );
+
+        MvcResult getCaseSubscriptionResponse =
+            mvc.perform(getSubscriptionByUuid(caseSubscriptionId))
+            .andExpect(status().isNotFound()).andReturn();
+        assertEquals(HttpStatus.NOT_FOUND.value(),
+                     getCaseSubscriptionResponse.getResponse().getStatus(),
+                     FORBIDDEN_STATUS_CODE
+        );
+        MvcResult getLocationSubscriptionResponse =
+            mvc.perform(getSubscriptionByUuid(locationSubscriptionId))
+            .andExpect(status().isNotFound()).andReturn();
+        assertEquals(HttpStatus.NOT_FOUND.value(),
+                     getLocationSubscriptionResponse.getResponse().getStatus(),
+                     FORBIDDEN_STATUS_CODE
```

Comment:
For 'not found' status code we can not throw FORBIDDEN_STATUS_CODE message. Either it should be specific to not found error message or more generic one like "Status codes do not match"

Reply:
fixed. change to "Status code does not match not found"